Cargo.lock

```diff
@@ -1,4 +1,99 @@
 [root]
 name = "battleship"
 version = "0.1.0"
+dependencies = [
+ "clap 2.20.1 (registry+https://github.com/rust-lang/crates.io-index)",
+]
 
+[[package]]
```

Comment:
Bei einem binary-project soll die `Cargo.lock` mit hochgeladen werden. http://doc.crates.io/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries